.github/workflows/main.yaml

```diff
@@ -9,16 +9,12 @@ on:
         description: Channel to post notifications
         default: build
         required: true
-      environment:
-        required: false
-        type: string
-        default: ''
```

Comment:
This is a breaking change requiring a new major release and a notice in the release notes about it.
Looks good to me, I assume you checked that no one uses the environment input here.

Alternatively, the default value could be `slack` to avoid any breaking changes and I think it would make quite sense here.

Up to you :+1:

Reply:
This is not a breaking change. I introduced the new variable in the last PR, https://github.com/SonarSource/gh-action_build-notify/pull/19, which has not been released yet.
Then, I realized that I could just hard-code it instead of passing it as an input. Thus it solves the original issue without requiring the user to pass the environment name from the workflow.